### PR TITLE
Copy Impl Fix

### PR DESCRIPTION
**Pull request by geomtech**
This pull request is ready to be merged.

All works, and I made a method to overwrite file like it already did in .NET

### CODE REVIEW

File: source/Cosmos.System2_Plugs/System/IO/FileImpl.cs [hunk 1]
```diff
@@ -162,11 +162,31 @@ public static void WriteAllBytes(string aFile, byte[] aBytes)
 
         public static void Copy(string srcFile, string destFile)
         {
-            using (var xFS = new FileStream(srcFile, FileMode.Open))
+            try
```
Comment (jp2masa) [inline source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: The `FileStream` code didn't work? I think it's faster to use `FileStream`.

File: source/Cosmos.System2_Plugs/System/IO/FileImpl.cs [hunk 2]
```diff
@@ -162,11 +162,31 @@ public static void WriteAllBytes(string aFile, byte[] aBytes)
...
+            }
+            catch (IOException ioEx)
+            {
+                throw new IOException("File Copy", ioEx);
+            }
+        }
+
+        public static void Copy(string srcFile, string destFile, bool overwriting)
+        {
+            if (overwriting)
```
Comment (jp2masa) [inline source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: If it's not overwriting, and the file doesn't exist, the copy has to be done.

File: source/Cosmos.System2_Plugs/System/IO/FileImpl.cs (hunk 1 above)
Comment (geomtech) [reply source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: No, it doesn't work.

File: source/Cosmos.System2_Plugs/System/IO/FileImpl.cs (hunk 2 above)
Comment (geomtech) [reply source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: Oh right ! I will modify this.

File: source/Cosmos.System2_Plugs/System/IO/FileImpl.cs (hunk 1 above)
Comment (geomtech) [reply source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: And the file created is empty

**Comment by geomtech** [comment]
I made a mistake, I open this pull with my Cosmos master branch lol, i recreate one

**Pull request closed without merging**